README.md

```diff
@@ -225,34 +225,34 @@ exposed is the logical one to follow in order to disclose different layers
 of complexity incrementally.
 
 Note: lately Redis was refactored quite a bit. Function names and file
-names changed, so you may find that this documentation reflects the
+names have been changed, so you may find that this documentation reflects the
 `unstable` branch more closely. For instance in Redis 3.0 the `server.c`
-and `server.h` files were renamed `redis.c` and `redis.h`. However the overall
+and `server.h` files were renamed to `redis.c` and `redis.h`. However the overall
```

Comment:
We should fix this sentence while we're at it. 
It should be `files were named ...`

(server.{c,h} are the _new_ names currently in testing/unstable)